package-lock.json

```diff
@@ -9,13 +9,24 @@
       "version": "3.0.12",
       "license": "MIT",
       "devDependencies": {
+        "change-case": "^5.4.3",
+        "emoji-platform-data": "^0.1.0",
         "fast-xml-parser": "^4.3.6",
         "prettier": "^1.18.2",
         "promptly": "^3.2.0",
         "tape": "^5.1.1",
         "unicode-emoji-json": "0.6.0"
       }
     },
+    "node_modules/@graphql-typed-document-node/core": {
```

Comment:
Aside: I don't love that adding `emoji-platform-data` as a dev dependency _also_ transitively adds in the packages `emojipedia` depends on for its [Node.js API](https://github.com/JoshuaKGoldberg/emojipedia/blob/a9e4c536e409df83a9fb34ffc02320c1e036c0cf/README.md#nodejs-apis). Seems silly. Tracked in https://github.com/JoshuaKGoldberg/emoji-platform-data/issues/200.